images/win/scripts/Tests/Vsix.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "Vsix" {
+    $toolset = Get-ToolsetContent
+    $requiredVsixs = $toolset.visualStudio.vsix
+
+    $allPackages = (Get-VSSetupInstance | Select-VsSetupInstance -Product *).Packages
+    $testCases = $requiredVsixs | ForEach-Object { @{ VsixId = $_.Id;  AllPackages = $allPackages }}
+
+    It "VSIX Extension <VsixId>" -TestCases $testCases {
+        $objVsix = $AllPackages | Where-Object {$_.id -eq $VsixId } | Select-Object Id, Version

```

Comment:
nitpick: `| Select-Object Id, Version` can be removed

Reply:
good point (it's the remainder of the previous logic).
Removed